py/janitor/site/simple.py

```diff
@@ -705,12 +705,18 @@ async def main_async(argv=None):
     public_runner = web.AppRunner(public_app)
     await private_runner.setup()
     await public_runner.setup()
+
     site = web.TCPSite(private_runner, args.host, port=args.port)
     await site.start()
-    logging.info("Listening on %s:%s", args.host, args.port)
+    logging.info("Private runner listening on %s:%s", args.host, args.port)
```

Comment:
This is not a runner

Reply:
That's the aiohttp server runner; not the same as a janitor runner.

Reply:
Whats the differences between public/private ports?